src/lib/sync-fs.js

```diff
@@ -80,12 +80,17 @@ function copy(from, to, { overwrite=true }={}) {
   }
 }
 
+function isDirectoryEmpty(dir) {
+  return fs.readdirSync(dir).length > 0 ? false : true;
```

Comment:
```suggestion
  return !fs.readdirSync(dir).length;
```

Minor, but no need to use a ternary to get a boolean value.